habana/tensorflow/buildspec-2-8-syai-1-3.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
We might also need to add the the older buildspec  for synAi 1.2 tf 2.7

Reply:
done